core/sqf/src/seatrans/tm/hbasetmlib2/src/main/java/org/trafodion/dtm/HBaseTxClient.java

```diff
@@ -452,7 +436,13 @@ public short abortTransaction(final long transactionID) throws Exception {
           Object commitDDLLock = new Object();
           synchronized(commitDDLLock)
           {
-            commitDDLLock.wait();
+             boolean loopBack = false;
+             try {
+                commitDDLLock.wait();
+             } catch(InterruptedException ie) {
+                 LOG.warn("Interrupting commitDDLLock.wait,  but retrying ", ie);
+                 loopBack = true;
+             } while (loopBack);
```

Comment:
This doesn't look right to me. Does "try" have a looping semantic when followed by "while"? Or did you intend to code a do { } around the try/catch block? (I'm thinking that the "while (loopBack);" does nothing if loopBack is false but loops infinitely if loopBack is true.)